quarkus.ls/com.redhat.quarkus.ls/src/main/java/com/redhat/quarkus/ls/client/CommandCapabilities.java

```diff
@@ -0,0 +1,101 @@
+/*******************************************************************************
+* Copyright (c) 2019 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package com.redhat.quarkus.ls.client;
```

Comment:
Sorry it was my fault, but please move those classes in the `com.redhat.quarkus.ls.commons.client` package. It should be a commons LSP feature.